docs/api/js/ethers-v6/getting-started.md

```diff
@@ -0,0 +1,92 @@
+---
+head:
+  - - meta
+    - name: "twitter:title"
+      content: JS SDK Getting Started | zkSync Era Docs
+---
+
+# Getting started
+
+::: info
+Smart contract and account deployment using hardhat plugin is under development.
+:::
+
+::: info
+Take a look at [migration guide](migration.md) if you are migration from `zksync-web3`.
+:::
+
+This is a short introduction to `zksync2-js` SDK, but covers many of the most common operations that developers require and provides a
+starting point for those newer to zkSync Era.
+
+## Getting zksync2-js
+
+### Prerequisites
+
+- Node: >=v18 ([installation guide](https://nodejs.org/en/download/package-manager))
+
+In order to install SDK for zkSync Era, run the command below in your terminal.
+
+```bash
+yarn add zksync2-js
+yarn add ethers@6 # ethers is a peer dependency of zksync2-js
+```
+
+## Overview
+
+While most of the existing SDKs should work out of the box, deploying smart contracts or using unique zkSync features, like account abstraction, requires providing additional
+fields to those that Ethereum transactions have by default.
+
+To begin, it is useful to have a basic understanding of the types of objects available and what they are responsible for, at a high level:
+
+- `Provider` provides connection to the zkSync Era blockchain, which allows querying the blockchain state, such as account, block or transaction details,
+  querying event logs or evaluating read-only code using call. Additionally, the client facilitates writing to the blockchain by sending
+  transactions.
+- `Wallet` wraps all operations that interact with an account. An account generally has a private key, which can be used to sign a variety of
+  types of payloads. It provides easy usage of the most common features.
+
+## Examples
+
+Connect to the zkSync Era network:
+
+```ts
+import { Provider, utils } from "zksync2-js";
```

Comment:
As a user, who is not native in the js/ts word, I would appreciate some explanation how these examples can be executed.

For example installing ts-node + `npx ts-node src/hello.ts`

Reply:
+ I think the `types` also should be imported here.

Reply:
`types` are imported. I also added following section:

## Examples

The following examples could be easily run by writing the code snippets in a file and executing them as follows:

```shell
npx ts-node src/zksync.ts
```